HW0/P6/P6B.py

```diff
@@ -16,16 +19,29 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    wait_time = 10**np.linspace(-6,0,20)
+
 
     for t in wait_time:
         # Compute jobs serially and in parallel
         # Use time.time() to compute the elapsed time for each
-        serialTime = 1
-        parallelTime = 1
+        timeBegining = time.time()
+
+        pool.map(burnTime, t*np.ones(N))
```

Comment:
This is a good way of calling 16 processes; I did it by 

for i in range(N/4): 
            pool.apply(burnTime, args=(t,))

I like your way better.